VERSION

```diff
@@ -1 +1 @@
-v0.7.3
+v0.7.4
```

Comment:
I merged https://github.com/antrea-io/ofnet/pull/61 so you should make this v0.8.1